src/main/java/Duke.java

```diff
@@ -10,6 +10,9 @@ public static void main(String[] args) {
         while (!isExit) {
             String input = in.nextLine();
             String command;
+            String description;
+            String by;
+            String from;
```

Comment:
consider using more descriptive variable names; instead of by, form, can use names like deadlineDate and eventStart